dev/jobs/prep.sh

```diff
@@ -130,6 +130,10 @@ if [[ ${err} -ne 0 ]]; then
    err_exit "JOBSPROC_GLOBAL_PREP job failed!"
 fi
 
+if [[ ! -f ${COMOUT_OBS}/${OPREFIX}prepbufr ]]; then
+   err_exit "The prepbufr file does not exist!"
```

Comment:
```suggestion
   export err=1
   err_exit "JOBSPROC_GLOBAL_PREP failed to create the prepbufr file, ABORT!"
```
Needs and `err` code to be set, I think.